actions/check.go

```diff
@@ -215,11 +216,15 @@ func Check(req CheckRequest) (*CheckResponse, error) {
       }
     }
 
-    // Only save the latest 
+    // Only save the latest
     if req.Source.When == "latest" && latestReviewIsMatch {
       versions = append(versions, *version)
     }
   }
 
+  sort.Slice(versions, func(i, j int) bool {
+    return versions[i].CommentID < versions[j].CommentID
+  })
```

Comment:
Hi! Thanks for opening this PR and taking the time at looking at this. :) 

Just a note here, in addition to comments, there are also ReviewIDs which need to be sorted

Reply:
How would you proceed?

From an API point of view, is a review a comment ? If it's the case, then as comment will be sorted, review will be too :thinking: